tests/model_inheritance/tests.py

```diff
@@ -187,7 +187,7 @@ def setUpTestData(cls):
             rating=2,
         )
 
-        chef = Chef.objects.create(name="Albert")
+        chef = Chef.objects.create(name="Albert", id=1)
```

Comment:
Assign `chef` to `cls` and use `self.chef.id` below instead as hardcoding pks might cause some issues.

Reply:
Better to put the test in `proxy_model_inheritance` so a new model isn't needed.